### DISCUSSION TITLE
how to trigger handover by other KPI

### DISCUSSION THREAD

**Discussion by natanzi** [srsran/srsRAN_Project#301] [General Help]
I have read the most recent [srsRAN documentation](https://buildmedia.readthedocs.org/media/pdf/srslte-docs/latest/srslte-docs.pdf) and I know that changing RR configuration files makes handovers easier. I am wondering if it is possible to implement handover scenarios by changing UE/cell/throughput, inducing congestion in a single cell, or changing other KPIs. 

Alternative handover triggers in srsRAN include:

- Loss of Synchronization (LOS)
- Loss of Signal (LOS)
- Excessive Bit Error Rate (BER)
- Poor Cell Range (PCR)
- Excessive Interference
- Radio Link Failure (RLF)

I would appreciate any insight you can provide.

**Comment by pgawlowicz** [comment]
Hi, you are referring to the srsRAN-4G documentation. Please ask you question here: https://github.com/srsran/srsRAN_4G
